---
bibliography: references.bib
output:
  pdf_document:
    keep_tex: true
    includes:
      in_header: header.tex
csl: mbio.csl
fontsize: 11pt
geometry: margin=1.0in
---

```{r knitr_settings, tidy=TRUE, eval=TRUE, echo=FALSE, cache=FALSE}
options(tidyverse.quiet = TRUE)

library(tidyverse)
library(knitr)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("message" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
  print(x)
  if(is.list(x)){
    x <- unlist(x)
  }
  if(is.numeric(x)){
    if(abs(x - round(x)) < .Machine$double.eps^0.5){
      paste(format(x,big.mark=',', digits=0, scientific=FALSE))
    } else {
      paste(format(x,big.mark=',', digits=2, nsmall=2, scientific=FALSE))
    }
  } else {
      paste(x)
  }
}
knitr::knit_hooks$set(inline=inline_hook)
```


\vspace*{10mm}

# Reintroducing mothur: 10 years later

\vspace{35mm}

Patrick D. Schloss${^1}$${^\dagger}$

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: \href{mailto:pschloss@umich.edu}{pschloss@umich.edu}

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109

\vspace{35mm}

## Observation format

\newpage
\linenumbers

## Abstract
75 words

\newpage

## Importance
150 words

\newpage

3000 words total


Goal: describe how to eat an elephant
mothur's origin story...
* Curvy thingy
* dotur/sons
* Sogin email
* ARB but for microbial ecology


Big points: identify the big problems, rate limiting steps, things that were just done wrong
* We may look back and shutter at how things were done, but we need some historical context
* If we solved all of the problems we would never publish or move the field forward
* The most important problem is answering your biological question
* Connect to Lenski's retrospective article

Competition is good and healthy
* A lot of people have been working in this space - e.g. clustering algorithms
* Approaches to developing software have differed (e.g. web-based (greengenes/RDP), boutique (Quince), wrappers (QIIME))


What would we have done differently?
* Develop mothur as an R package - but tools weren't available, better graphics
* Testing suite from the beginning
* Better job of engaging community


What are we proud of?
* Open source packages
* Development of open instructional materials
* Data driven development of methods
* Database independent - allows people outside of our field to use mothur


Helped create standards
* When something is this popular, you force people to use a certain suite of tools and methods


The future
* 16S rRNA gene sequencing is not dead
* Always improving sequencing methods bring new problems
* Larger datasets, need a replacement for naive open and closed reference clustering
* Lingering controversies - singletons, ASVs

Longterm funding








\newpage

## Acknowledgements


\newpage

## References
25 references

<div id="refs"></div>

\newpage

<!--- \includegraphics{figure_1.png} -->

**Figure 1. Caption caption caption.** Footnotes footnotes footnotes.
